components.js

```diff
@@ -1,20 +1,8 @@
 import * as React from 'react'
 import * as ReactDOM from 'react-dom'
 
-// export class ShapeView extends React.Component {
-//   render() {
-//     return
-//   }
-// }
 let count = 0;
 
-// export var ShapeView = React.createClass({
-//  render: function() {
-//    return <div>
-//      {this.props.shape.squares().map(sq => <Square row={sq.row} col={sq.col} />)}
-//    </div>;
-//  }
-// });
 export var GameView = React.createClass({
```

Comment:
It is generally a better idea to break up components into different files, even if the lines of code are few. Makes your code more modular and the codebase easier to navigate.

Also, `export const` if you don't plan on reinitializing the variable.